packages/graphql/src/translate/create-auth-and-params.ts

```diff
@@ -293,7 +294,7 @@ function createAuthAndParams({
                 predicateParams = { ...predicateParams, ...par };
             });
 
-            thisPredicates.push(predicates.join(` ${key} `));
+            thisPredicates.push(joinPredicates(predicates, key as "AND" | "OR"));
```

Comment:
Like before, instead of just casting, I think it is worth it changing how we get the key to ensure it is properly typed.
In this case, instead of just looping through a string array
```
l.272  ["AND", "OR"].forEach((key) => {
```

We could explicitly set the types:
```ts
var keys:Array<"AND"|"OR"> = ["AND", "OR"]
keys.forEach((key)=>{
```

Reply:
Makes sense to me me